test/performance/cpu/parameters.yml

```diff
@@ -0,0 +1,70 @@
+receivers: "cpu"
```

Comment:
This PR only covers a single plugin instead of the entire host resource usage plugin (e.g mem). The following PR will have additional use case for performance test.